src/components/List/SimpleList/SimpleList.story.jsx

```diff
@@ -0,0 +1,320 @@
+import React from 'react';
```

Comment:
Can we limit the amount of stories needed here with the use of Knobs?

Reply:
@davidicus do we have a ideal range for # stories?

Reply:
< 5. I think if stories are needed we should show the functionality but we also do not want stories to get lost like they do for components like the Table or valuecards which have huge list of stories to go through and you can not easily make out what they are. 

For these stories I can not tell the difference between the last two. Are both needed?